app/resources/admin.html.template

```diff
@@ -39,17 +39,18 @@
 <h2>Select a repository</h2>
 
 <script>
-  var LANGUAGE_EXONYMS = {{language_exonyms_json|safe}};
 
-  /* Returns true if the array contains elem */
-  function arr_contains(arr, elem) {
-    for (var i = 0; i < this.length; i++) {
-      if (arr[i] === elem) {
-        return true;
-      }
-    }
-    return false;
-  }
+//TODO: move to object-oriented!!
+
+  var LANGUAGE_EXONYMS = {{language_exonyms_json|safe}};
+  var CUSTOM_MESSAGE_TABLE_IDS = [
```

Comment:
repo_titles is not a custom message, so naming this "CUSTOM_MESSAGE_TABLE_IDS" is a bit weird. Can you rename it to e.g., "MULTILINGUAL_TABLE_NAMES"?